### PR TITLE
Graph is reder base on code attribute. Real data from BD for code 1 w…

### PR DESCRIPTION
**Pull request by cazapatamar**
…hich is transformed before render.

### CODE REVIEW

File: src/Indicator.jsx [hunk 1]
```diff
@@ -21,6 +21,8 @@ class Indicator extends React.Component {
       biomesList: [],
       connError: false,
       data: null,
+      code: 1,
```
Comment (erikasv) [inline src/Indicator.jsx]: since in GraphData the defaults correspond to those related with code 4, we should use 4 instead of 1

**Review by erikasv — DISMISSED** [review]
After the change you can merge, if you need any help with the conflicts just tell me

File: src/Indicator.jsx (hunk 1 above)
Comment (cazapatamar) [reply src/Indicator.jsx]: Done

**Review by erikasv — APPROVED** [review]
Please merge, the problem with the graph is related to react-google-charts dependency.

**Pull request merged**